src/js/rtc_peer_connection_factory.js

```diff
@@ -35,7 +35,7 @@ export default class RtcPeerConnectionFactory {
     }
 
     _isEarlyMediaConnectionSupported() {
-        this._strategy._isEarlyMediaConnectionSupported();
+        return this._strategy._isEarlyMediaConnectionSupported();
```

Comment:
Not blocking this commit but think on how we could have avoided this - Unit tests, Metrics monitoring early peer connections?

Reply:
Yeah, we should have both unit test and metrics.